src/display/node_stream.js

```diff
@@ -28,14 +28,30 @@ import {
 
 const fileUriRegex = /^file:\/\/\/[a-zA-Z]:\//;
 
+function parseUrl(sourceUrl) {
+   var parsedUrl = url.parse(sourceUrl);
```

Comment:
Please make sure that the indenting in this function is two spaces, not three. Also, use `let` instead of `var`.